roofit/roofitcore/inc/RooMinimizer.h

```diff
@@ -44,26 +44,49 @@ class RooMinimizer : public TObject {
 public:
    enum class FcnMode { classic, gradient, generic_wrapper };
 
+   /// Config argument to RooMinimizer ctor
+   struct Config {
+      double recoverFromNaN = 10.; // RooAbsMinimizerFcn config
+      int optConst = 0;            // RooAbsMinimizerFcn config
+      int printEvalErrors = 10;    // RooAbsMinimizerFcn config
+      int doEEWall = 1;            // RooAbsMinimizerFcn config
+      int offsetting = -1;         // RooAbsMinimizerFcn config
+      const char *logf = nullptr;  // RooAbsMinimizerFcn config
+      int nWorkers = getDefaultWorkers(); // RooAbsMinimizerFcn config that can only be set in ctor
+      int parallelGradient = 0;          // RooAbsMinimizerFcn config that can only be set in ctor
+      int parallelLikelihood = 0;        // RooAbsMinimizerFcn config that can only be set in ctor
+      int verbose = 0;                    // local config
+      bool profile;                       // local config
```

Comment:
It should also have a default, otherwise the value is not undefined when the struct is initialized:
`bool profile = false;` (I guess)